src/apps/applib/lib/utility.c

```diff
@@ -242,7 +242,8 @@ void randomGeo(GeoCoord* g) {
 int countActualHexagons(H3Index* hexagons, int numHexagons) {
     int actualNumHexagons = 0;
     for (int i = 0; i < numHexagons; i++) {
-        if (hexagons[i] != H3_INVALID_INDEX) {
+        // should we actually use `h3IsValid` here?
```

Comment:
In a tight loop that would be expensive. Since this function is not exported, we can know *all* callsites and be sure that it only returns valid indexes or zero.

Reply:
got it. removed the comment!